packages/slo-stat-bot/src/slo-stat-label.ts

```diff
@@ -0,0 +1,239 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {Application, Context} from 'probot';
+import {GitHubAPI} from 'probot/lib/github';
+import Ajv, {ErrorObject} from 'ajv';
+
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const schema = require('./../data/schema.json');
+
+type Conclusion =
+  | 'success'
+  | 'failure'
+  | 'neutral'
+  | 'cancelled'
+  | 'timed_out'
+  | 'action_required'
+  | undefined;
+
+type ValidationResults = {
+  isValid: boolean;
+  errors?: ErrorObject[] | null;
+};
+
+type PullsListFilesResponseItem = {
+  filename: string;
+  sha: string;
+};
+
+function handler(app: Application) {
+  app.on(
+    [
+      'pull_request.opened',
+      'pull_request.reopened',
+      'pull_request.edited',
+      'pull_request.synchronize',
+    ],
+    async (context: Context) => {
+      const owner = context.payload.repository.owner.login;
+      const repo = context.payload.repository.name;
+      const pull_number = context.payload.number;
+
+      const fileList = await handler.listFiles(
+        context.github,
+        owner,
+        repo,
+        pull_number,
+        100
+      );
+
+      if (!fileList) {
+        return;
+      }
+
+      for (const file of fileList) {
+        //Checks to see if file is repo level or org level issue_slo_rules.json
+        if (
+          file.filename === '.github/issue_slo_rules.json' ||
```

Comment:
@JustinBeckwith @orthros, do we think that this will be where we move some of the sots of configuration that previously lived in sloth?